docs/content/component-status/components/Statuses.js

```diff
@@ -4,23 +4,23 @@ import Badge from '../../../../src/components/Badge';
 import { circuit } from '../../../../src/themes';
 
 const Stable = () => (
-  <ThemeProvider theme={circuit}>
+  <ThemeProvider theme={{ ...circuit }}>
     <Badge color={Badge.SUCCESS} style={{ display: 'inline-block' }}>
       Stable
     </Badge>
   </ThemeProvider>
 );
 
 const Deprecated = () => (
-  <ThemeProvider theme={circuit}>
+  <ThemeProvider theme={{ ...circuit }}>
     <Badge color={Badge.DANGER} style={{ display: 'inline-block' }}>
       Depcreated
     </Badge>
   </ThemeProvider>
 );
 
 const InReview = () => (
-  <ThemeProvider theme={circuit}>
+  <ThemeProvider theme={{ ...circuit }}>
```

Comment:
I'm guessing I need to adjust this import.

Reply:
You can import directly from the main export (`src/index.js`). Theme are already transformed there.